### PR TITLE
[HPCMI-AD] Cypress tests

### PR DESCRIPTION
**Pull request by charmcitygavin**
## Purpose

Adds Cypress tests for HPCMI-AD

#### Linked Issues to Close

_Links to issue(s) that are closed by this PR. Be sure to use the phrase "Closes #XXX" for each issue, so they automatically close_

## Approach

_How does this change address the issue?_

## Learning

_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Assorted Notes/Considerations

_List any other information that you think is important... a post-merge activity, someone to notify, etc._

#### Pull Request Creator Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [ ] Someone has been assigned this PR.
- [ ] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -83,49 +83,50 @@ jobs:
...
-            fuaADoy2_8940.spec.feature,
             fuaADoy2_15377.spec.feature,
+            fuaADoy2_8940.spec.feature,
+            fuhch.spec.js,
+            hpcmi-ad.spec.js,
             lbwCH.spec.feature,
             lrcdCH.spec.feature,
             nciddsAD.spec.feature,
-            oy2_7763_FAQ_Page_State_View.spec.js,
-            oy2_8977_MSC_AD.spec.js,
```
Comment (aalousi) [inline .github/workflows/deploy.yml:99]: just wondering what are those changes in the deploy.yml?

**Review by aalousi — APPROVED** [review]
everything else looks great!

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (charmcitygavin) [reply .github/workflows/deploy.yml:99]: I sorted this list alphabetically to improve readability. The spec on line 98, for example, is now on 115. (Don't worry, nothing was removed!) We've been sorting things alphabetically in other parts of the code and I figured it wouldn't hurt to be consistent here. 🙂

Comment (aalousi) [reply .github/workflows/deploy.yml:99]: Awesome, thanks for the clarification!

Comment (charmcitygavin) [reply .github/workflows/deploy.yml:99]: Any time! 😎

File: tests/cypress/support/commands/commands.js [hunk 2]
```diff
@@ -191,3 +191,10 @@ Cypress.Commands.add("clickValidateMeasure", (timeout = 500) => {
   cy.get('[data-cy="Validate Measure"]').click();
   cy.wait(timeout);
 });
+
+Cypress.Commands.add("enterValidDateRange", () => {
+  cy.get('[data-cy="DateRange.startDate-month"]').type("1");
+  cy.get('[data-cy="DateRange.startDate-year"]').type("2021");
+  cy.get('[data-cy="DateRange.endDate-month"]').type("12");
+  cy.get('[data-cy="DateRange.endDate-year"]').type("2021");
+});
```
Comment (benjaminpaige) [inline tests/cypress/support/commands/commands.js:200]: 🔥 🔥 🔥 🔥

**Review by benjaminpaige — APPROVED** [review]
love the enterValidDateRange command

**Pull request merged**